source/codegen/metadata/nirfmxspecan_restricted/config_addon.py

```diff
@@ -0,0 +1,5 @@
+# We need to maintain the version here since it needs to be updated by the build process on GitHub
+config_additional_config = {
+    'module_version': '1.3.3.dev0',
+    'latest_runtime_version_tested_against': '21.3.0',
```

Comment:
Is this supposed to be the driver version we tested against? Is the version accurate? Or is there a convention that we should we be testing against a specific older version?

Reply:
I copied these over from the corresponding nirfmxspecan files. On my machine, I ran my tests with the latest 22.8 dev builds of the specan driver. But I am not sure what version the automated test pipeline is setup to use.